src/GoProControl.cpp

```diff
@@ -400,12 +399,31 @@ char *GoProControl::getStatus()
   if (_camera == HERO3)
   {
     makeRequest(_request, "/camera/sx?t=", _pwd);
+	sendHTTPRequest(_request);
```

Comment:
the code can't go here, otherwise sendHTTPRequest is called twice

Reply:
I modified it, I will try if it still work